### PR TITLE
Dls 813

### PR DESCRIPTION
**Pull request by JackieDev**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -57,7 +57,7 @@ play.modules.enabled += "uk.gov.hmrc.helptosave.modules.EligibilityStatsModule"
 # Global request handler
 # ~~~~
 # Set to the object handler for migrated 2.3 services
-play.http.requestHandler = "play.api.http.GlobalSettingsHttpRequestHandler"
+#play.http.requestHandler = "play.api.http.GlobalSettingsHttpRequestHandler"
```
Comment (lutnos) [inline conf/application.conf]: Better to remove that section than to comment a line, I think

Comment (Jackie1988) [reply conf/application.conf]: Think that may be a mistake, I forgot to put it back after commenting it out. Will put it back in and see what happens.

Comment (Jackie1988) [reply conf/application.conf]: It looks like the GlobalSettingsHttpRequestHandler has disappeared so I have updated it to JavaCompatibleHttpRequestHandler which extends DefaultHttpRequestHandler. https://github.com/playframework/playframework/issues/5357

Comment (Jackie1988) [reply conf/application.conf]: I realize the above link is a bit old (Dec 2015)

**Pull request closed without merging**